src/BlsData.jl

```diff
@@ -6,6 +6,7 @@ using Requests
 using DataFrames
 import JSON
 import HttpCommon
+using Compat
```

Comment:
this part can't be done retroactively, so might be more reliable to re-tag for it. also see https://github.com/attobot/attobot for a useful tagging tool